server/index.js

```diff
@@ -5,18 +5,50 @@ const path = require ('path');
 var cors = require ('cors');
 
 app.use (cors ());
+app.use(express.json());
 app.use (express.static (path.join (__dirname + '/../public')));
 
-app.get (`/artist/:id/relatedArtists`, (req, res) => {
+// https://stackoverflow.com/questions/411462/restful-way-to-create-multiple-items-in-one-request
+// https://stackoverflow.com/questions/32098423/rest-updating-multiple-resources-with-one-request-is-it-standard-or-to-be-avo
+// https://softwareengineering.stackexchange.com/questions/191596/is-it-ok-to-partially-change-a-collection-with-put-or-delete
+
+// CREATE
+app.post('/artists/:artistId/related-artists', (req, res) => {
```

Comment:
reminder to use /api/v1/artists...

Reply:
Great catch, thanks for the reminder. Fixed 👍